src/components/gas/GasSpeedLabelPager.js

```diff
@@ -44,7 +44,7 @@ const GasSpeedLabelPager = ({
 
   return (
     <Row align="center" height={GasSpeedLabelPagerItemHeight} justify="end">
-      {showPager && (
+      {showPager && options?.length > 1 && (
```

Comment:
Note to self: question about not having paging in
- SwapDetailsState
- TransactionConfirmationScreen
- SendSheet for non-optimism